.idea/misc.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we want these or should we .gitignore the .idea folder? I think the main issue was that IntelliJ and PyCharm have incompatible formats for this folder. I'd tend towards ignoring this and writing a PyCharm/IntelliJ projects setup section in the README.md.